### PR TITLE
Updated with missing documentation

### PR DESCRIPTION
**Pull request by muraliQlogic**
Fixes #<issue_number_goes_here> (it's a good idea to open an issue first for discussion)

- [ ] Tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

### CODE REVIEW

File: src/filter.js [hunk 1]
```diff
@@ -222,6 +222,9 @@ Filter.parse = function(filters) {
 };
 
 /**
+ * Sets pallAllFilter or blockAllFilter
+ *
+ * @param {boolean} pass Whether to pallAllFilter or blockAllFilter
```
[the remark this answers is not in the document]
Comment (muraliQlogic) [reply src/filter.js]: Comments in example already explains the meaning of true/false . Do you want me to add explanation in param description?

[the remark this answers is not in the document]
Comment (sduskis) [reply src/filter.js]: yes please. Something like "true for passAllFilter, false for blockAllFilter" should be an easy and informative change.

[the remark this answers is not in the document]
Comment (muraliQlogic) [reply src/filter.js]: Updated as suggested.

File: src/filter.js [hunk 2]
```diff
@@ -535,21 +544,22 @@ Filter.prototype.label = function(label) {
...
- * // Note that, since row keys can contain arbitrary bytes, the '\C' escape
- * // sequence must be used if a true wildcard is desired. The '.' character
- * // will not match the new line character '\n', which may be present in a
- * // binary key.
- * //
- * // In the following example we'll use a regular expression to match all
- * // row keys ending with the letters "on", which would then yield
- * // `gwashington` and `tjefferson`.
- * //-
+ * In the following example we'll use a regular expression to match all
```
[the remark this answers is not in the document]
Comment (muraliQlogic) [reply src/filter.js]: Updated as suggested.

**Pull request merged**